README.md

```diff
@@ -78,37 +78,70 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 
 
 ## Usage
-### Model preparation 
+Imagine you want to train a `LinearRegression` model based on your dataset `X`, `y`. After splitting the data into train/test segments you would have `X_train` and `y_train` as your training data. So you train your model as follows.
 ```pycon
->>> from sklearn import datasets
->>> from pymilo import Export, Import
 >>> from sklearn.linear_model import LinearRegression
->>> import os
->>> X, Y = datasets.load_diabetes(return_X_y=True)
->>> threshold = 20
->>> X_train, X_test = X[:-threshold], X[-threshold:]
->>> Y_train, Y_test = Y[:-threshold], Y[-threshold:]
 >>> model = LinearRegression()
->>> #### Train the model using the training sets
 >>> model.fit(X_train, Y_train)
```

Comment:
y_train

Reply:
What a coincidence!
I just told him in Telegram :))